frontend/src/App.tsx

```diff
@@ -1,19 +1,47 @@
 import "./App.css";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 import Input from "./components/Input";
 import TodoList from "./components/TodoList";
 import { Todo } from "./models/todo.model";
 
+const urlString = "http://localhost:3001/todos/";
+
 function App() {
   const [todos, setTodos] = useState<Todo[]>([]);
   const [newTodoDescription, setNewTodoDescription] = useState("");
+  const [loading, setLoading] = useState(true);
 
-  const onChange = (name: string, value: string) => {
-    setNewTodoDescription(value);
-  };
+  function fetchTodos(url: string, signal: AbortSignal) {
```

Comment:
You are using a mixture of regular functions and arrow functions. Using either is fine, but you have to be consistent. Either declare all functions with the `function` keyword, or make them all arrow functions.